### PR TITLE
Add Telegram docs

### PR DESCRIPTION
**Pull request by ivov**
Here are the Telegram docs!

The new example workflow is [here](https://n8n.io/workflows/451). I also added bot credentials instructions.

Let me know if you have any comments!

### CODE REVIEW

File: docs/nodes/nodes-library/nodes/Telegram/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	- Send a text message
+	- Send a group of photos or videos to an album
+	- Send a photo
+	- Send a sticker
+	- Send a video
+
+## Example Usage
+
+This workflow shows you how to send a message to a Telegram channel. You can also find the [workflow](https://n8n.io/workflows/451) on this website. This example usage workflow uses the following two nodes.
+- [Start](../../core-nodes/Start)
```
Comment (Shraddha2104) [inline docs/nodes/nodes-library/nodes/Telegram/README.md]: i guess it should be  `../../core-nodes/Start/README.md` , according to the feedback received on one of my docs

File: docs/nodes/nodes-library/nodes/Telegram/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+![A workflow with the Telegram node](./workflow.png)
+
+### 1. Start node
+
+The Start node exists by default when you create a new workflow.
+
+### 2. Telegram node
+
+1. Create a credential as shown [here](../../../credentials/Telegram/).
+4. Select *Message* in the *Resource* field.
```
Comment (Shraddha2104) [inline docs/nodes/nodes-library/nodes/Telegram/README.md]: ordering needs to be fixed

**Comment by ivov** [comment]
Thanks @Shraddha2104! Fixes applied.

**Comment by ivov** [comment]
Submitted to maintainers: https://github.com/n8n-io/n8n-docs/pull/27

**Pull request closed without merging**